Python/Tests/Core/CondaEnvironmentManagerTests.cs

```diff
@@ -112,30 +112,6 @@ public class CondaEnvironmentManagerTests {
             AssertCondaMetaFiles(envPath, "python-2.7.*.json");
         }
 
-        [TestMethod, Priority(0)]
```

Comment:
Create additional tests to replace this one?

Reply:
I've removed the test because conda changed behavior between 4.5.11 and later versions. Rather than fail when attempting to create an environment that already exists, they now delete+create. So I figure it's not worth it anymore.

Reply:
So, if a user accidentally creates a new environment with the same name as existing environment, their previous environment would be deleted. Do we give them some kind of warning or confirmation option to prevent them from accidentally doing this?

Reply:
Add environment dialog checks for invalid or conflicting names, and doesn't allow you to proceed with creation if you specify an already existing name.